src/shared/domain/repositories/action_repository_interface.py

```diff
@@ -8,4 +9,7 @@ class IActionRepository(ABC):
     @abstractmethod
     def get_member(self, ra: str) -> Member:
         pass
-    
\ No newline at end of file
+    
+    @abstractmethod
+    def get_all_actions_by_ra(self, ra: str) -> List[Action]:
```

Comment:
coloca isso daqui "return [] if member doesn't have actions"

Comentário serve para, quando for fazer o método do bd real, lembrar de algumas características pequenas, mas importantes.

Reply:
ok